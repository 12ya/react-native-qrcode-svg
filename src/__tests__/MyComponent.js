// https://github.com/facebook/react/issues/7386
jest.mock('react/lib/ReactComponentEnvironment');
import 'react-native';
import React from 'react';
import QRCode from '../index';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <QRCode />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
