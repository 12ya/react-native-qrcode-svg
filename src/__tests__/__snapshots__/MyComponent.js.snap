exports[`test renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Rect
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Path
  </Text>
</View>
`;
